src/test/enums.spec.js

```diff
@@ -86,51 +93,23 @@ struct MyStruct {
       // language=JavaScript
       'index.js': `
 // @flow
```

Comment:
Would it make sense to move these fixtures to their own files for easier tracking?